src/plugins/validation/2and3/semantic-validators/pagination-ibm.js

```diff
@@ -0,0 +1,181 @@
+module.exports.validate = function({ jsSpec }, config) {
+  const result = {};
+  result.error = [];
+  result.warning = [];
+
+  config = config.pagination;
+  const queryParameters = [];
+  for (const head in jsSpec.paths) {
+    if (/{/.test(head)) {
+      continue;
+    }
+    if (jsSpec.paths[head].get) {
+      for (const resp in jsSpec.paths[head].get.responses) {
+        for (const con in jsSpec.paths[head].get.responses[resp].content) {
+          if (con == 'application/json') {
+            let arrayOnTop = false;
+            if (
+              (jsSpec.paths[head].get.responses[resp].content[con].schema &&
+                jsSpec.paths[head].get.responses[resp].content[con].schema
+                  .properties) ||
+              (jsSpec.paths[head].get.responses[resp].content[con].schema
+                .pagination &&
+                jsSpec.paths[head].get.responses[resp].content[con].schema
+                  .pagination.properties)
```

Comment:
Schema will generally not have `pagination` as a property.  If it is present, it would appear in `schema.properties`.